### PR TITLE
Added jobname variable.

### PR DESCRIPTION
**Pull request by vinjana**

### CODE REVIEW

File: src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy [hunk 1]
```diff
@@ -374,6 +374,42 @@ class LSFJobManager extends BatchEuphoriaJobManagerAdapter {
...
+        return 'LSB_QUEUE'
+    }
+
+    @Override
+    String getJobArrayIndexVariable() {
+        return "LSB_JOBINDEX"
+    }
+
+    @Override
+    String getNodeFileVariable() {
```
Comment (dankwart-de) [inline src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy]: What is a node file?

File: src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy [hunk 2]
```diff
@@ -374,6 +374,42 @@ class LSFJobManager extends BatchEuphoriaJobManagerAdapter {
...
+        return "LSB_JOBNAME"
+    }
+
+    @Override
+    String getQueueVariable() {
+        return 'LSB_QUEUE'
+    }
+
+    @Override
+    String getJobArrayIndexVariable() {
```
Comment (dankwart-de) [inline src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy]: Thought we want to get rid of this?

File: src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy [hunk 3]
```diff
@@ -118,6 +118,11 @@ class DirectSynchronousExecutionJobManager extends BatchEuphoriaJobManager<Direc
     }
 
     @Override
+    String getJobNameVariable() {
```
Comment (dankwart-de) [inline src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy]: $ is supposed to return a job name? How is this supposed to work?

Comment (vinjana) [reply src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectSynchronousExecutionJobManager.groovy]: For the DirectSynchronousExecutionJobManager there is no good name for a process. So return the process ID ($$ in bash).

File: src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy (hunk 1 above)
Comment (vinjana) [reply src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy]: Here I added variables from PBS. Could be that not all are important. This is a candidate for a less important one. I can remove it. I actually decided not to include all variables anyway, becaus not all variables can trivially be mapped semantically between PBS and LSF (not checked SGE and SLURM -- probably similar) and some have no reasonably equivalent (also for direct execution).

**Comment by vinjana** [comment]
I kept the node file variable as it has no further implications on our code. For the array jobs, we decided to remove the support from the library (for now) and therefore I removed also the variable.

**Pull request merged**